src/encoded/types/experiment.py

```diff
@@ -382,29 +411,50 @@ def superseded_by(self, request, superseded_by):
         "type": "string"
     })
     def replication_type(self, request, replicates=None, assay_term_name=None):
+        # ENCD-4251 loop through replicates and select one replicate, which has
+        # the smallest technical_replicate_number, per biological replicate.
+        # That replicate should have a libraries property which, as calculated
+        # in replicate.libraries (ENCD-4251), should have collected all
+        # possible technique replicates belong to the biological replicate.
+        # TODO: change this once we remove technical_replicate_number.
+        bio_rep_dict = {}
+        for rep in replicates:
+            replicateObject = request.embed(rep, '@@object')
```

Comment:
Does this have to be camelCase?

Reply:
No. I tried to keep it consistent with other properties, like biosample_summary, here and consistent with old code. Any suggestions?

Reply:
Let's break this pattern if you can make your property self-consistent.